docs/en/reference/advanced-configuration.rst

```diff
@@ -33,18 +33,7 @@ steps of configuration.
     $config->setMetadataDriverImpl($driverImpl);
     $config->setQueryCache($queryCache);
 
-    if (PHP_VERSION_ID > 80400) {
-        $config->enableNativeLazyObjects(true);
-    } else {
-        $config->setProxyDir('/path/to/myproject/lib/MyProject/Proxies');
-        $config->setProxyNamespace('MyProject\Proxies');
-
-        if ($applicationMode === "development") {
-            $config->setAutoGenerateProxyClasses(true);
-        } else {
-            $config->setAutoGenerateProxyClasses(false);
-        }
```

Comment:
All of this code is now dead-ish: time to kill it?

Reply:
@Ocramius now is the time: https://github.com/doctrine/orm/pull/12029 :slightly_smiling_face:

Reply:
Almost nostalgic :O